lib/cassandra/utils.rb

```diff
@@ -1,3 +1,4 @@
 require_relative 'utils/cli/base'
 require_relative 'utils/daemon'
 require_relative 'utils/stats'
+require_relative 'utils/autoclean'
```

Comment:
I thought we could stick things that do something to the system in a directory like `tasks`.